NAMING.transientNameServerBadPort="IOP00110600: (BAD_PARAM) Port 0 is not a valid port in the transient name server"
NAMING.transientNameServerBadHost="IOP00110601: (BAD_PARAM) A null hostname is not a valid hostname in the transient name server"
NAMING.objectIsNull="IOP00110602: (BAD_PARAM) Invalid object reference passed in rebind or bind operation"
NAMING.insBadAddress="IOP00110603: (BAD_PARAM) Bad host address in -ORBInitDef"
NAMING.bindUpdateContextFailed="IOP00010600: (UNKNOWN) Updated context failed for bind"
NAMING.bindFailure="IOP00010601: (UNKNOWN) bind failure"
NAMING.resolveConversionFailure="IOP00010602: (UNKNOWN) Resolve conversion failed"
NAMING.resolveFailure="IOP00010603: (UNKNOWN) Resolve failure"
NAMING.unbindFailure="IOP00010604: (UNKNOWN) Unbind failure"
NAMING.transNsCannotCreateInitialNcSys="IOP00910650: (INITIALIZE) SystemException in transient name service while initializing"
NAMING.transNsCannotCreateInitialNc="IOP00910651: (INITIALIZE) Java exception in transient name service while initializing"
NAMING.namingCtxRebindAlreadyBound="IOP00710600: (INTERNAL) Unexpected AlreadyBound exception iun rebind"
NAMING.namingCtxRebindctxAlreadyBound="IOP00710601: (INTERNAL) Unexpected AlreadyBound exception in rebind_context"
NAMING.namingCtxBadBindingtype="IOP00710602: (INTERNAL) Bad binding type in internal binding implementation"
NAMING.namingCtxResolveCannotNarrowToCtx="IOP00710603: (INTERNAL) Object reference that is not CosNaming::NamingContext bound as a context"
NAMING.namingCtxBindingIteratorCreate="IOP00710604: (INTERNAL) Error in creating POA for BindingIterator"
NAMING.transNcBindAlreadyBound="IOP00710700: (INTERNAL) Bind implementation encountered a previous bind"
NAMING.transNcListGotExc="IOP00710701: (INTERNAL) list operation caught an unexpected Java exception while creating list iterator"
NAMING.transNcNewctxGotExc="IOP00710702: (INTERNAL) new_context operation caught an unexpected Java excpetion creating the NewContext servant"
NAMING.transNcDestroyGotExc="IOP00710703: (INTERNAL) Destroy operation caught a Java exception while disconnecting from ORB"
NAMING.insBadSchemeName="IOP00710705: (INTERNAL) Stringified object reference with unknown protocol specified"
NAMING.insBadSchemeSpecificPart="IOP00710707: (INTERNAL) Malformed URL in -ORBInitDef"
NAMING.insOther="IOP00710708: (INTERNAL) Malformed URL in -ORBInitDef"
